components/SummaryStats/SummaryStats.tsx

```diff
@@ -53,14 +57,25 @@ function Row({ label, value }: IRow) {
 
 export default function SummaryStats() {
   const summaryStats = useSummaryStats()
+  const symbol = useTickerSelection()
+  const socket = useContext(SocketIOContext)
+
+  useEffect(() => {
```

Comment:
You'd want to handle this in a stream too ->

`tickerSelections$.pipe(switchMap(symbol => sendSocketMessageObservable(symbol)))`